### PR TITLE
Change Several Roletimers

### PR DESCRIPTION
**Pull request by luckyshotpictures**
## About the PR

This changes most command roles to be three hours of overall playtime, and warden to one hour.

## Why / Balance

Discussion amongst GMs concluded that letting new players settle into Umbra via a shift played as a non-command role would be preferable to hopping into a Command position immediately due to the higher standards for Command characters imposed by the HRP setting.

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

### CODE REVIEW

File: Resources/Prototypes/Roles/Jobs/Command/captain.yml [hunk 1]
```diff
@@ -4,18 +4,8 @@
...
-      department: Medical
-      time: 54000 # 15 hours
-    - !type:DepartmentTimeRequirement
-      department: Security
-      time: 54000 # 15 hours
-    - !type:DepartmentTimeRequirement
-      department: Command
-      time: 54000 # 15 hours
+      department: OverallPlaytimeRequirement
+      time: 10800 # 3 hours
```
Comment (Simyon264) [inline Resources/Prototypes/Roles/Jobs/Command/captain.yml:8]:
```suggestion
    - !type:OverallPlaytimeRequirement
      time: 10800 # 3 hours
```

File: Resources/Prototypes/Roles/Jobs/Command/centcom_official.yml [hunk 2]
```diff
@@ -25,5 +25,3 @@
     id: CentcomPDA
     ears: ClothingHeadsetAltCentCom
     belt: WeaponPistolN1984
-    pocket1: BoxFolderBlack
-    pocket2: PenCentcom
```
Comment (Simyon264) [inline Resources/Prototypes/Roles/Jobs/Command/centcom_official.yml:29]: This has nothing to do with role timers. Why is this change in this PR?

File: Resources/Prototypes/Roles/Jobs/Command/head_of_personnel.yml [hunk 3]
```diff
@@ -4,18 +4,8 @@
...
-      department: Medical
-      time: 36000 # 10 hours
-    - !type:DepartmentTimeRequirement
-      department: Security
-      time: 36000 # 10 hrs
-    - !type:DepartmentTimeRequirement
-      department: Command
-      time: 36000 # 10 hours
+      department: OverallPlaytimeRequirement
+      time: 10800 # 3 hours
```
Comment (Simyon264) [inline Resources/Prototypes/Roles/Jobs/Command/head_of_personnel.yml:8]:
```suggestion
    - !type:OverallPlaytimeRequirement
      time: 10800 # 3 hours
```

File: Resources/Prototypes/Roles/Jobs/Science/research_director.yml [hunk 4]
```diff
@@ -5,10 +5,8 @@
   playTimeTracker: JobResearchDirector
   requirements:
     - !type:DepartmentTimeRequirement
```
Comment (Simyon264) [inline Resources/Prototypes/Roles/Jobs/Science/research_director.yml:7]:
```suggestion

```

**Review by Simyon264 — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Roles/Jobs/Science/research_director.yml (hunk 4 above)
Comment (Simyon264) [reply Resources/Prototypes/Roles/Jobs/Science/research_director.yml:7]: You removed the playtime requirements for the department but left in the type for it.

File: Resources/Prototypes/Roles/Jobs/Security/warden.yml [hunk 5]
```diff
@@ -5,8 +5,8 @@
   playTimeTracker: JobWarden
   requirements:
     - !type:RoleTimeRequirement
```
Comment (TsjipTsjip) [inline Resources/Prototypes/Roles/Jobs/Security/warden.yml:7]: `- !type:OverallPlaytimeRequirement`

**Review by Simyon264 — APPROVED** [review]
YAML LGTM

File: Resources/Prototypes/Roles/Jobs/Command/centcom_official.yml (hunk 2 above)
Comment (TsjipTsjip) [reply Resources/Prototypes/Roles/Jobs/Command/centcom_official.yml:29]: Change is too minor to be atomized into its own PR at this time.

**Pull request merged**